src/components/wasd-to-analog2d.js

```diff
@@ -3,6 +3,9 @@
  * @namespace user-input
  * @component wasd-to-analog2d
  */
+
+const WASD = ["w", "a", "s", "d"];
+
 AFRAME.registerComponent("wasd-to-analog2d", {
```

Comment:
We can remove this component entirely. It is a relic from `aframe-input-mapping-component` that should have been removed long ago. Nothing fires any of the events (in `eventNames`) that it binds to.